src/pages/home/report/ReportActionItem.js

```diff
@@ -78,6 +99,7 @@ class ReportActionItem extends Component {
             || this.props.hasOutstandingIOU !== nextProps.hasOutstandingIOU
             || this.props.shouldDisplayNewIndicator !== nextProps.shouldDisplayNewIndicator
             || !_.isEqual(this.props.action, nextProps.action)
+            || !_.isEqual(this.props.shouldHighlight, nextProps.shouldHighlight)
             || this.state.isContextMenuActive !== nextState.isContextMenuActive;
     }
```

Comment:
i don't see `this.props.action.reportActionID` here, shouldn't it also be included?